### PR TITLE
Added Java code snippets for cuckoo filters

### PR DESCRIPTION
**Pull request by Ranjeet0611**
Added Java code snippets for cuckoo filters #3631

### CODE REVIEW

**Comment by Ranjeet0611** [comment]
Hi @dwdougherty Please review this one

File: src/test/java/io/redis/examples/CuckooFilterExample.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+public class CuckooFilterExample {
+    @Test
+    public void run(){
+        UnifiedJedis jedis = new UnifiedJedis("redis://localhost:6379");
+        // HIDE_END
+
+        // REMOVE_START
+        jedis.del("bikes:models");
+        // REMOVE_END
+
```
Comment (dwdougherty) [inline src/test/java/io/redis/examples/CuckooFilterExample.java:19]: Please a // STEP_START line here.

Comment (Ranjeet0611) [reply src/test/java/io/redis/examples/CuckooFilterExample.java:19]: Hi @dwdougherty Added thank you

**Review by dwdougherty — APPROVED** [review]
LGTM. Docs build correctly.

A Java person should reiew this. @sazzad16 ?

**Comment by sazzad16** [comment]
@dwdougherty Could you go through other PRs from [this list](https://github.com/redis/jedis/pulls?q=is%3Apr+is%3Aopen+label%3Adocs-review)?

**Comment by dwdougherty** [comment]
I will do, @sazzad16. I'll try to get to it this week as soon as I wrap up a large project.

**Comment by dwdougherty** [comment]
@sazzad16 This PR builds correctly and it's good to go.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**